tests/assets/experiment/config_artifact.yaml

```diff
@@ -32,7 +32,6 @@ features_config:
   use_sentinel_bands:
   - B02
   - SCL
-last_commit_hash: ea440f57d459c98cf7cc03f7e24a0df05c5e23d2
```

Comment:
When I run `make assets`, I get a `config_artifact.yaml` that is written out with `last_commit_hash` included.

`make assets` calls `cyfi/experiment.py`, so the hash is still written out when we re-create the assets even if re-running the tests doesn't write out the hash or change the `config_artifact.yaml` test asset. Is this expected? Or do we want to be able to rerun `make assets` without changing the asset files in addition to running `make tests` without changing the files?

Reply:
great flag, i forgot we had `make assets`.

> Or do we want to be able to rerun make assets without changing the asset files in addition to running make tests without changing the files?

we do want this. i've put in a better fix that uses a temp dir as the save location for the test since we really shouldn't let tests make persistent changes

it's fine that `make assets` writes out the git hash since that lets us track provenance of when assets were created